server/socket.js

```diff
@@ -16,7 +17,7 @@ value = {
 }
 */
 var players = {};
-var nextRoom = 0;
+var nextRoom = 1; // start with a truthy room name
```

Comment:
Only because having a falsy room name made the first line of code in line 27 (inside `getOpponentSocket`) kinda messy. I would have to make it `room = room === undefined ? socket.rooms[1] : room` and I think this was a simple solution to make it all cleaner. I could also make all room names strings, but makes it harder to increment.